setup.py

```diff
@@ -290,12 +275,12 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
     'nose-timer',
     'parameterized',
     'paramiko',
-    'pylint~=2.3.1',  # Ensure the same version as in .travis.yml
+    'pylint',
```

Comment:
Do we want to un-pin this. Does pylint have a habit of adding new checks in? (We've been stung by flake8 adding new checks before)

Reply:
it's a proability indeed. I will revert it and we should anyhow work out better way of dealing with dependencies - so we might want to get some auto-upgrades. I switched to dependabot in our other project and it works nicely (though it's agains Apache rules).